.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 100,
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "none"
+}
```

Comment:
Are these personal preferences or are they inherited from related projects?

Reply:
@fazouane-marouane Inherited from other projects under the unifiedjs umbrella. 
I can update this if you notice any drift already. If not, we can merge it and modify it in the future.

Reply:
Let’s try this! We can change it later if need be.